dev

```diff
@@ -43,4 +43,4 @@ fi
 # build and run dev.ts
 # tsc is configured to build what we expect in tsconfig.json
 pnpm run -r build:dev 
-node ./dev_tool/build/dev.js "$@"
+node ./dev_tool/build/src/dev.js "$@"
```

Comment:
@mojotalantikite We updated `dev_tool/tsconfig.json` `include` configuration to include `eslient.config.ts` to the build. This moved all the dev tool code to `build/src/` so updated the dev script to run from there. Let me know if this isn't the way we want to do this.

cc: @pearl-truss